packages/path_provider/path_provider_linux/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 2.0.0-nullsafety.1
```

Comment:
You'll need to do an actual point release for this since they already landed.

Reply:
done